src/pages/MapViewer/components/Toolbox/components/item-assets/AssetsPanel.tsx

```diff
@@ -0,0 +1,49 @@
+import { ClipboardButton } from '@/components/clipboard/ClipboardButton';
+import { useToolbox } from '@/hooks/useToolbox';
+
+import './AssetsPanel.scss';
+
+export const AssetsPanel = () => {
+  const {
+    state: { selectedCollectionItem },
+    actions: { setActivePage },
+  } = useToolbox();
+
+  // Reduce the object of objects down to an array of key/value objects.
+  const assets = Object.keys(selectedCollectionItem.assets).reduce((acc, key) => {
+    acc = [...acc, { key, value: selectedCollectionItem.assets[key].href }];
+
+    return acc;
+  }, []);
+
+  return (
+    <div>
+      <div className="toolbox__header">
+        <button
+          className="button-link"
+          onClick={() => {
```

Comment:
I've seen that you would typically shorten this to:
``onClick={() => setActivePage('items')}``

Reply:
good catch, I'll simplify. I tend to write it out like this first then remove the curly braces once ready to commit, obviously I missed this out here.